runtime/src/snapshot_utils.rs

```diff
@@ -465,8 +465,18 @@ fn delete_contents_of_path(path: impl AsRef<Path>) {
     }
 }
 
+/// Deletes the contents of a directory asynchronously to avoid blocking on it,
+/// but leaves the top directory in place.
+pub fn move_and_async_delete_path_contents(path: impl AsRef<Path>) {
+    move_and_async_delete_path(&path);
+    // The following could fail if the rename failed.
+    // If that happens, the directory should be left as is.
+    // So we ignore errors here.
+    let _ = std::fs::create_dir(path);
```

Comment:
pretty sure this will create a new inode entry and any open handles to the old one will remain valid.  not sure if that matters here, but is at least an inconsistency with the documented behavior

Reply:
I'll update the comment to make it clear it's actually (move + async delete) then re-creating the directory.

Function is used at start up to clean accounts directories quickly, but we still want those directories to exist.